run.go

```diff
@@ -445,6 +450,14 @@ func runAction(clicontext *cli.Context) error {
 		opts = append(opts, privilegedOpts...)
 	}
 
```

Comment:
> Signed-off-by: rudeigerc

Could you use your real name, LGTM then.

Thanks.

Reply:
Updated. Thanks a lot.